docs/docs/setup/auth/jwt.md

```diff
@@ -74,7 +74,7 @@ A few points and suggestions regarding the configuration:
   ```xml
   <property name="usernameMapping" value="['username'].asString()" />
   <property name="nameMapping" value="['name'].asString()" />
-  <property name="emailMapping" value="['username'].asString() + '@tobira.invalid'" />
+  <property name="emailMapping" value="['email'].asString()" />
```

Comment:
Might be worth adding `<!-- Or if you don't have emails for your user: ['username'].asString() + '@tobira.invalid' -->